cypress/support/cypress-support/src/main.js

```diff
@@ -126,8 +126,14 @@ export default function (module) {
 
   // beforeEach
   beforeEach(() => {
+    UTILS.getEnvVariable(CONSTANTS.FB_INTERACTIONLOGS).clearLogs();
     cy.getBeforeOperationObject();
     UTILS.destroyGlobalObjects([CONSTANTS.LIFECYCLE_APP_OBJECT_LIST]);
+    let scenarioName = cy.state().test.title;
+    if (scenarioName.includes('(example')) {
```

Comment:
Reading the code, it is unclear why we would need to modify the scenario name when `example` exists. Can we add comments to make it clear.

Reply:
Removed the above code and using it from before operation object